app/README.md

```diff
@@ -1,49 +1,48 @@
-# radicle desktop client
+## App
 
-This is a cross-platform desktop client for Radicle.
-We currently support Linux and macOS.
+A frontend written in JavaScript using Svelte and Electron.
 
 
-## Development
+### Development
 
-The app is written in Svelte + Electron. For our dependency management and
-script execution we use `yarn`.
+For dependency management and script execution we use `yarn`. Code formatting
+is dictated by [prettier][pr] and enforced locally on a pre-commit basis with
+[husky][hu] and [lint-staged][ls].
 
-Code formatting is dictated by [prettier][pr] and we enforce it locally on
-a pre-commit basis with [husky][hu] and [lint-staged][ls].
 
+#### Running the app
 
-### Setup
+1. Get the code: `git clone git@github.com:radicle-dev/radicle-upstream.git`.
+2. Install dependencies: `cd radicle-upstream/app && yarn install`.
+3. Start the app in development mode: `yarn start`.
 
-Run the app locally with hot reloading:
 
-1. Get the code: `git clone git@github.com:radicle-dev/radicle-upstream.git`
-2. Install dependencies: `cd radicle-upstream/app && yarn install`
-3. Start the app in development mode: `yarn start`
+#### Running tests
 
+Before running tests locally you'll need to set up a test fixture repository:
+`git submodule foreach "git fetch --all"`.
 
-Build and package the app:
-
-1. Build: `yarn dist`
-2. Get the generated package from: `dist/radicle-upstream-0.0.1.dmg`
-
+- to run the tests do: `yarn test`
+- to troubleshoot tests in the Cypress GUI, run: `yarn test:debug`
```

Comment:
Should we explain how to run a single test?

Reply:
:+1: